SpatialGDK/Source/SpatialGDK/Private/Tests/MockObject.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Utils/RPCContainer.h"
+
+#include "Core.h"
+#include "MockObject.generated.h"
+
+using namespace SpatialGDK;
```

Comment:
Might need to be careful putting a using namespace in a header.

Reply:
Removed using namespace